DOCUMENTATION.md

```diff
@@ -361,11 +369,13 @@ If the request is not successful, the client returns an `Alphagov\Notifications\
 
 ### Prerequisites
 
-When your service first signs up to GOV.UK Notify, you’ll start in [trial mode](https://www.notifications.service.gov.uk/features/using-notify#trial-mode). You can only send letters in live mode. You must ask GOV.UK Notify to make your service live.
+When you add a new service it will start in [trial mode](https://www.notifications.service.gov.uk/features/trial-mode). You can only send letters when your service is live.
 
-1. Sign in to [GOV.UK Notify](https://www.notifications.service.gov.uk/).
-1. Select __Using Notify__.
-1. Select __requesting to go live__.
+To send Notify a request to go live:
```

Comment:
Here is another use of Notify without GOV.UK. Just to be clear, I don't have a strong opinion on whether this is best or not but just wanted to point out a potential inconsistency that I'll leave you to make a decision on.

Reply:
The rule we use on the app is:

>Use GOV.UK Notify the first time you mention the product on a page. After that you can call it Notify.

The adaptation of this rule Jon and I discussed for the docs is to use the full name the first time it's mentioned in a new section, then it's at our discretion to drop the 'GOV.UK' if Notify is mentioned several times in context.

This looks like an example of that.

It's probably fine for now, but we may revisit in the future if the rule doesn't work for docs the same way it does for each page of the app.